docs/src/integrate/nodejs-api.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Node.js API
+title: Node.js API Reference
 eleventyNavigation:
     key: node.js api
-    parent: developer guide
-    title: Node.js API
-    order: 9
+    parent: extend eslint
```

Comment:
Are we moving this now or later?

```suggestion
    parent: integrate eslint
```

Reply:
later! integrate section doesn't exist yet. i'd like to get a bit more content in the section before moving, and want to keep this PR scoped specifically to the TOC updates.